inbm/dispatcher-agent/dispatcher/aota/aota_command.py

```diff
@@ -50,6 +50,11 @@ def __init__(self,
                  dbs: ConfigDbs) -> None:
         # security assumption: parsed_manifest is already validated
 
+        if 'signature' in parsed_manifest and parsed_manifest['signature'] is not None:
```

Comment:
Consider initializing `_signature` and `_hash_algorithm` to `None` to ensure they have default values if not provided in the manifest. [important]